### PR TITLE
[mp-7-wip] WFLY-19592 Update MicroProfile Fault Tolerance Subsystem documentatio…

### PR DESCRIPTION
**Pull request by rhusar**
…n section.

Community documentation for https://issues.redhat.com/browse/WFLY-19592

Docs only - so no CI needed.

### CODE REVIEW

**Comment by rhusar** [comment]
@fabiobrz was asked to review, thanks!

File: docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Fault_Tolerance_SmallRye.adoc [hunk 1]
```diff
@@ -34,13 +34,16 @@ The MicroProfile Fault Tolerance implementation is provided by the https://githu
...
-however, it is not included by default in the default configuration of WildFly.
+however, it is not included in the default configuration of WildFly.
 
-NOTE: The MicroProfile Metrics extension and subsystem are required by this extension to provide Metrics integration,
-please follow the instructions in the <<required-extension-microprofile-metrics-smallrye,MicroProfile Metrics Subsystem Configuration section>>.
-If the Metrics subsystem is not available, no metrics data will be collected.
+This extension requires the MicroProfile Telemetry extension and subsystem
+to provide metrics collection integration as defined by the specification.
+Please follow the instructions in the <<MicroProfile_Telemetry,MicroProfile Telemetry Subsystem section>>.
+If the MicroProfile telemetry subsystem is not available, no metrics data will be collected.
```
Comment (fabiobrz) [inline docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Fault_Tolerance_SmallRye.adoc:42]: I'd suggest s/no metrics data/no fault-tolerance related metrics data/ - to make it clear we're relating to  the one the spec integration would produce. E.g.: the server itself would be able to generate metrics, either natively or via Micrometer.

WDYT?

**Review by fabiobrz — COMMENTED** [review]
Thanks for the PR @rhusar - I've added just one comment.

File: docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Fault_Tolerance_SmallRye.adoc (hunk 1 above)
Comment (rhusar) [reply docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Fault_Tolerance_SmallRye.adoc:42]: I think that's obvious, but OK changed!

**Pull request merged**